### DISCUSSION TITLE
Remove all styling?

### DISCUSSION THREAD

**Discussion by royduin** [appbaseio/reactivesearch#1553] [Ideas]
Is it possible to remove all styling and style all elements with my own classes? So remove the injected styles from emotion completely.

**Comment by bietkul** [comment]
@royduin Removal is not possible however you can customize ReactiveSearch components in the following ways
- You can define your class at the component level to override the default style.
- Most of the components support `render` and `renderItem` methods that allow you to use your own UI elements.
- `ReactiveComponent` can be used to integrate any UI component to Elasticsearch.

**Reply by royduin** [reply]
Thanks for your answer, maybe transform this question to a feature request; it would be nice if there was an option to disable all styles so no emotion styles are injected. Classes may stay.
